expressions/expressions.y

```diff
@@ -43,7 +44,17 @@ func init() {
 %left '<' '>'
 %%
 start:
-  cond ';' { yylex.(*lexer).val = $1 }
+  cond ';' {
+	  yylex.(*lexer).val =  func(ctx Context) values.Value { 
+		retVal := $1(ctx)
+		switch retVal.(type) {
+			case values.ArrayValue:
+				return values.ValueOf(strings.Join(retVal.Interface().([]string), ", "))
```

Comment:
I'm afraid this is gonna panic if the retValue is slice of other types like `[]interface{true, false}`?

Reply:
I'll test it with the unit test, indeed I'm not sure it will work for every array value right now. But I think (I'm not sure) that what they are doing with cycle3 in the yaml file is converting array to a string array